src/components/Layout/OverviewList.tsx

```diff
@@ -40,6 +43,15 @@ export default React.memo(function OverviewList(props: OverviewListProps) {
     dataPlace,
   } = props
   const l = useFormatMessage()
+
+  const isBigScreen = useMediaQuery({ minWidth: 1800 })
+
+  const cardsToShow = useCardsByWidth({
+    cardWidth: isBigScreen ? 310 : 260,
+    cardMargin: 14,
+    containerMargin: 48,
+  })
```

Comment:
This looks too hardcoded 😅 , Isn't any other way to get these values directly from the components? That way, we can eventually create an standard component that handles all this nice behavior

Reply:
I removed the isBigDesktop and set a card fixed width in useCardsByWidth hook and also into the css.
Is this good enough or should I extract those numbers (cardWidth, cardMargin, containerMargin) dynamically using useRef?

Reply:
We should detect it dynamically, but it is not necessarily urgent. We can merge it if there is and issue to fix it later.